.idea/.name

```diff
@@ -0,0 +1 @@
+TheMovieDB
```

Comment:
Vẫn chưa remove ra khỏi dù đã list trong .gitignore rồi này e
@vietnh-1109 support bạn hộ anh với

https://stackoverflow.com/questions/13541615/how-to-remove-files-that-are-listed-in-the-gitignore-but-still-on-the-repositor